src/provisioning/Provisioner.ts

```diff
@@ -860,9 +870,15 @@ export class Provisioner extends ProvisioningApi {
 
         const ircDomain = body.remote_room_server;
         const ircChannel = body.remote_room_channel;
+
+
```

Comment:
Nitpick, is one newline sufficient? :slightly_smiling_face:

Reply:
It looks like I do more work! But yep, removed.